### PR TITLE
UI : removed uncessary buttons and improve UI

### PR DESCRIPTION
**Pull request by sahsisunny**
**Description:**

This pull request focuses on improving the user interface by simplifying the navigation and removing unnecessary elements from the dashboard page. The changes made in this PR include:

1. **Navbar Update**: Removed unnecessary options from the navigation bar to streamline the user experience.

2. **Dashboard Page Cleanup**: Removed redundant buttons from the dashboard page, enhancing clarity and reducing clutter.
3. **Config**: Changes related to snapshots have been made, such as disabling snapshot generation and removing the snapshot generation test.

### Screenshot

**Before**
image

**After**
image

**Toast**
image

**Test Cases**
image

### CODE REVIEW

**Comment by vinit717** [comment]
please mention test stats for pages/dashboard/index

File: src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -57,21 +57,11 @@ const Navbar: React.FC = () => {
...
                                 Dashboard
                             </a>
                         </li>
-                        <li>
-                            <a href="#" className="text-white hover:bg-gray-700 block px-4 py-2">
-                                Settings
-                            </a>
-                        </li>
                         <li>
                             <a href={TINY_API_LOGOUT} className="text-white hover:bg-gray-700 block px-4 py-2">
```
Comment (gauravsinhaweb) [inline src/components/Navbar/index.tsx:66]: make an array of nav items and map it, If it is repetitive. This will make nav items dynamic and scalable

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]
- Add an interactive copy to clipboard functionality, in the dashboard page
- Improve Navbar test case stats

File: src/components/Navbar/index.tsx (hunk 1 above)
Comment (sahsisunny) [reply src/components/Navbar/index.tsx:66]: I don't think using this approach is the best option for only two elements.

Comment (sahsisunny) [reply src/components/Navbar/index.tsx:66]: There is a minor performance overhead when using `map` for a small number of items

**Comment by sahsisunny** [comment]
> * Improve Navbar test case stats

We have 10-11 files that need improvement for testing. Additionally, this PR made no changes to the Navbar line showing uncovered.

**Comment by gauravsinhaweb** [comment]
> > * Improve Navbar test case stats
> 
> We have 10-11 files that need improvement for testing. Additionally, this PR made no changes to the Navbar line showing uncovered.

comparing it to previous test results of the navbar, it has differences.

**Comment by gauravsinhaweb** [comment]
> * Add an interactive copy to clipboard functionality, in the dashboard page
> * Improve Navbar test case stats

- just make it generic, add green tick on click
- if not link, disable interaction

**Comment by sahsisunny** [comment]
> > > * Improve Navbar test case stats
> > 
> > 
> > We have 10-11 files that need improvement for testing. Additionally, this PR made no changes to the Navbar line showing uncovered.
> 
> comparing it to previous test results of the navbar, it has differences.

Where? Can you share the stats?

**Comment by sahsisunny** [comment]
> > * Add an interactive copy to clipboard functionality, in the dashboard page
> > * Improve Navbar test case stats
> 
> * just make it generic, add green tick on click
> * if not link, disable interaction

Toast is added

**Comment by gauravsinhaweb** [comment]
> > > > * Improve Navbar test case stats
> > > 
> > > 
> > > We have 10-11 files that need improvement for testing. Additionally, this PR made no changes to the Navbar line showing uncovered.
> > 
> > 
> > comparing it to previous test results of the navbar, it has differences.
> 
> Where? Can you share the stats?

my bad, better if navbar stats could be improved

**Review by gauravsinhaweb — APPROVED** [review]

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]
the url should not be generated if there is no link provided by the user, add the tests acoordingly

**Comment by sahsisunny** [comment]
> the url should not be generated if there is no link provided by the user, add the tests acoordingly

Feature is not implemented to generate short URL

**Comment by sahsisunny** [comment]
> > > > > * Improve Navbar test case stats
> > > > 
> > > > 
> > > > We have 10-11 files that need improvement for testing. Additionally, this PR made no changes to the Navbar line showing uncovered.
> > > 
> > > 
> > > comparing it to previous test results of the navbar, it has differences.
> > 
> > 
> > Where? Can you share the stats?
> 
> my bad, better if navbar stats could be improved

You haven't been active for 6 days. Can you improve test coverage for all files?

**Comment by gauravsinhaweb** [comment]
> > > > > > * Improve Navbar test case stats
> > > > > 
> > > > > 
> > > > > We have 10-11 files that need improvement for testing. Additionally, this PR made no changes to the Navbar line showing uncovered.
> > > > 
> > > > 
> > > > comparing it to previous test results of the navbar, it has differences.
> > > 
> > > 
> > > Where? Can you share the stats?
> > 
> > 
> > my bad, better if navbar stats could be improved
> 
> You haven't been active for 6 days. Can you improve test coverage for all files? image

No

**Review by gauravsinhaweb — APPROVED** [review]

**Comment by gauravsinhaweb** [comment]
> > the url should not be generated if there is no link provided by the user, add the tests acoordingly
> 
> Feature is not implemented to generate short URL

I don't think if it is dependent on feature, if the there is no input length, don't call the link generator fn

File: src/pages/dashboard/index.tsx
Comment (vinit717) [inline src/pages/dashboard/index.tsx:1]: Please mention test stats for this

**Review by vinit717 — APPROVED** [review]

**Pull request merged**